suzieq/poller/worker/services/igmp.py

```diff
@@ -0,0 +1,99 @@
+import re
+from datetime import datetime
+
+from dateparser import parse
+import numpy as np
+
+import json
+
+from suzieq.poller.worker.services.service import Service
+
+
+
+class IgmpService(Service):
+    """Igmp Service."""
+
+    def clean_json_input(self, data):
+        """Junos JSON data for some older ver needs some work"""
+        pass
```

Comment:
Why are you overriding the default function provided in service.py?

Reply:
Placeholder here. Will remove.